packages/firestore/scripts/run-tests.ts

```diff
@@ -57,8 +58,9 @@ let args = [
 
 if (argv.emulator) {
   process.env.FIRESTORE_EMULATOR_PORT = '8080';
-  process.env.FIRESTORE_EMULATOR_PROJECT_ID = 'test-emulator';
+  process.env.FIRESTORE_EMULATOR_PROJECT_ID = EMULATOR_PROJECT_ID;
 }
+export const ALT_PROJECT_ID = 'test-db2';
```

Comment:
This seems slightly off. Who is importing this file?

Reply:
Removed.